pkg/kubelet/dockertools/manager.go

```diff
@@ -62,6 +62,8 @@ const (
 
 	MinimumDockerAPIVersion = "1.18"
 
+	OOMScoreAdjDockerAPIVersion = "1.21"
```

Comment:
nit: This version adds a few more features. Can you rename this to be `dockerv110APIVersion` instead?